internal/helpers/proxy/proxy.go

```diff
@@ -98,11 +104,16 @@ func DetermineProxyTypeForInstance(conf *config.Config, dbin *kindav1beta1.DbIns
 	switch backend {
 	case "google":
```

Comment:
Google instances are deprecated, we shouldn't modify it

Reply:
done